Library/OcVariableLib/VariableManagement.c

```diff
@@ -416,14 +415,51 @@ GetBootstrapBootData (
   return OptionData;
 }
 
+OC_FIRMWARE_RUNTIME_PROTOCOL *
+OcDisableFirmwareRuntime (
+  VOID
+  )
+{
+  EFI_STATUS                    Status;
+  OC_FIRMWARE_RUNTIME_PROTOCOL  *FwRuntime;
+  OC_FWRT_CONFIG                Config;
+
+  Status = gBS->LocateProtocol (
+                  &gOcFirmwareRuntimeProtocolGuid,
+                  NULL,
+                  (VOID **)&FwRuntime
+                  );
+
+  if (!EFI_ERROR (Status) && (FwRuntime->Revision == OC_FIRMWARE_RUNTIME_REVISION)) {
+    ZeroMem (&Config, sizeof (Config));
+    FwRuntime->SetOverride (&Config);
+    DEBUG ((DEBUG_INFO, "OCVAR: Found FW NVRAM, full access %d\n", Config.BootVariableRedirect));
+  } else {
```

Comment:
How about do the error checking first, then return NULL immediately?

Reply:
@PMheart - This code was like this already (just moved, and deduplicated) - I'm not sure this comment is correct, it already does return NULL immediately if LocateProtocol returns an error, no?

Reply:
Okay, leaving this one as is after discussion over PM.